README.md

```diff
@@ -0,0 +1,164 @@
+
+## Community
+
+If you want to get in contact with the **Terasology** community and the whole **MovingBlocks** team, you can easily connect with us, share your ideas, report and solve problems.
+We are present in nearly the complete round-up of social networks. Follow/friend us wherever you want, chat with us and tell the world.
+
+&nbsp;
+
+<p align="center">
+    <a title="Discord" href="https://discord.gg/terasology">
+        <img src="./docs/images/discord.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Twitter" href="https://twitter.com/Terasology">
+    <img src="./docs/images/twitter.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Facebook" href="https://www.facebook.com/Terasology">
+        <img src="./docs/images/facebook.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Reddit" href="http://www.reddit.com/r/Terasology">
+        <img src="./docs/images/reddit.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Youtube" href="https://www.youtube.com/user/blockmaniaTV">
+        <img src="./docs/images/youtube.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Patreon" href="https://www.patreon.com/Terasology">
+        <img src="./docs/images/patreon.jpg" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Terasology Forum" href="https://forum.terasology.org">
+        <img src="./docs/images/forum.png" width="48px"/>
+    </a>
+</p>
+
+
+## Installation
+
+<table>
+    <tr>
+        <td></td>
+        <th>Minimum Requirements</th>
+    </tr>
+    <tr>
+        <td>System (OS)</td>
+        <td>Windows, MacOS, Linux (64 bit)</td>
+    </tr>
+    <tr>
+        <td>Processor (CPU)</td>
+        <td>dual-core CPU</td>
+    </tr>
+    <tr>
+        <td>Memory (RAM)</td>
+        <td>2 GB</td>
+    </tr>
+    <tr>
+        <td>Graphics (GPU)</td>
+        <td style="vertical-align:top">
+            Intel HD Graphics (Gen 7)<br/>
+            GeForce 8xxx series (or higher) or<br/>
+            Radeon HD 2000 series (or higher)<br/>
+            with OpenGL 3.3
+        </td>
+    </tr>
+    <tr>
+        <td>Storage (HDD)</td>
+        <td>1 GB</td>
+    </tr>
+</table>
+
+Internet connectivity is required for downloading Terasology via the Launcher, afterwards playing offline is possible.
+
+For easy game setup (recommended) you can use our launcher - [download it here](https://terasology.org/download.html).
+
+For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules](docs/Modules.md).
+
+
+### Alternative Installation Methods
+
+If you already have Java Runtime Environment (JRE) installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java version 11 is supported.
```

Comment:
```suggestion
If you already have Java Runtime Environment (JRE) installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java version 11 is required.
```

Reply:
JRE is not exists since java 9 releases. Only jdk releases present.

Another note: launcher using javafx, which shipping in some openjdk releases and not shing in another..

Reply:
> JRE is not exists since java 9 releases. Only jdk releases present.
Thanks for the hint regarding JRE vs JDK.

> Another note: launcher using javafx, which shipping in some openjdk releases and not shing in another..
For _running_ the launcher, we don't need a jdk or javafx as it's self-contained. You'll only need this for _developing_ for the launcher, which is not subject of this README. It is mentioned in the MovingBlocks/TerasologyLauncher README, though.

Reply:
@Cervator 's suggestion - done
@DarkWeird correction regarding JRE vs JDK - done